graphicstext/c01-introduction/s3.xml

```diff
@@ -14,7 +14,7 @@ you a bit of background about the history of OpenGL and about the graphics
 hardware that supports it.</p1>
 
 <p>In the first desktop computers, the contents of the screen were managed
-directly by the <word>CPU</word>.  For example, to draw a line segment on the screen, the CPU
+directly by the <newword>CPU</newword>.  For example, to draw a line segment on the screen, the CPU
```

Comment:
I cloned the repo and searched for uses of CPU, API, and vertex. I believe these are the first time they are used in the book.

Reply:
I'm probably not very consistent about how &lt;word> and &lt;newword> are used, but &lt;newword> is really meant for when a definition of the word is given in the text, so not exactly for the first occurrence of the word in the book.

Reply:
I reverted this changes to using `<newword>`. For upcoming PRs, I won't make changes to these tags.